SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -195,14 +195,15 @@ void USpatialReceiver::HandlePlayerLifecycleAuthority(Worker_AuthorityChangeOp&
 		{
 			if (USpatialNetConnection* Connection = Cast<USpatialNetConnection>(PlayerController->GetNetConnection()))
 			{
+				EntityConnectionMap.Add(Op.entity_id, Connection);
```

Comment:
I feel like the name of this map is a bit misleading. Since the PlayerController is checked out, we have created a connection for it, so if I naively saw this map, I would expect the player controller and connection to be in here even if I don't have authority. Potentially rename or maybe change logic to check for authority later?